src/models/threads.rs

```diff
@@ -42,6 +42,8 @@ pub enum MessageBody {
         #[serde(default)]
         private: bool,
         replying_to: Option<ThreadMessageId>,
+        #[serde(default)]
+        associated_images: Vec<ImageId>,
```

Comment:
what's the point of this field? shouldn't you be able to get this from the thread message id

Reply:
the thread message post route takes a MessageBody as part of the posting (this enum). Since ::Text is the only one with images attached, it made sense to put it in here. I can move it out to the surrounding web::Json though if youd prefer it out there

(ie: this is primarily for the creation of it, not for the getting of it, there just isnt a builder like there is for proj/versions)

Reply:
would it be needed on creation? iirc the route to attach images to something is a separate route right?

Reply:
resolving with discord convo